app/views/events/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1 class="main-title">Events list</h1>
+<main>
+  <div class="left-side">
+    <h2>Past events</h2>
+    <% Event.past.each do |event| %>
+      <p>
+        <%= link_to event.title, event_path(event.id) %>
+        by <%= link_to event.creator.username, user_path(event.creator.id) %>
+        
+      </p>
+    <% end %>
+  </div>
+
+  <div class="right-side">
+    <h2>Upcoming events</h2>
+    <% Event.upcoming.each do |event| %>
```

Comment:
We can also have:
```ruby
@future_events = Event.upcoming
```
in the index action of events controller and just use:
```ruby
@future_event.each do |event|
```
in the views